src/unix/notbsd/linux/mod.rs

```diff
@@ -657,6 +549,169 @@ s! {
     }
 }
 
+s_no_extra_traits!{
```

Comment:
One could manually derive the traits for these, right? Might want to add a TODO here.